decidim-conferences/config/locales/en.yml

```
@@ -321,7 +321,9 @@ en:
         deleted_conferences_info: Conferences can only be deleted if status is "Unpublished".
     admin_log:
       conference:
+        duplicate: "%{user_name} duplicated the %{resource_name} conference"
         create: "%{user_name} created the %{resource_name} conference"
+        duplicate: "%{user_name} duplicated the %{resource_name} conference"
```

YAMLlint (1.38.0)

[error] duplication of key "duplicate" in mapping

(key-duplicates)